pylintrc

```diff
@@ -209,7 +209,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=20
+max-public-methods=22
```

Comment:
`TestGoogleStyleGuideChecker` now has 21 public methods; this class should be refactored in a future PR and this max lowered (https://github.com/Shopify/shopify_python/issues/84).